data/reusables/identity-and-permissions/team-sync-usage-limits.md

```diff
@@ -1,6 +1,6 @@
 ### Usage limits
 
-There are usage limits for the team synchonization feature.  Exceeding these limits will lead to a degredation in performance and may cause synchronization failures.
+There are usage limits for the team synchronization feature.  Exceeding these limits will lead to a degradation in performance and may cause synchronization failures.
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/managing-team-synchronization-for-your-organization#usage-limits
<img width="696" alt="image" src="https://github.com/github/docs/assets/2119212/147b5c64-3ca4-4b78-8669-27f12d849453">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/managing-team-synchronization-for-your-organization#usage-limits
<img width="705" alt="image" src="https://github.com/github/docs/assets/2119212/ac94b762-cec1-4174-9fa3-6945d6a55c3f">